src/cli/load.js

```diff
@@ -122,6 +128,14 @@ export function makeLoadCommand(
           explicitOutput = stringToRepoId(args[i]);
           break;
         }
+        case "--weights": {
+          if (weightsPath != undefined)
```

Comment:
In general an optional field `weightsPath?: string` doesn't match nulls so I used undefined here.